bangazonreports/views/products/expensiveproducts.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+from django.shortcuts import render
+from bangazonapi.models import Product
+from bangazonreports.views import Connection
+
+
+def expensive_list(request):
+    if request.method == 'GET':
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+
+            db_cursor.execute("""
+                select
+                    p.id,
+                    p.name,
+                    p.description,
+                    p.price
+                from bangazonapi_product p
+                where p.price > 999
```

Comment:
What if the price is 999.5? That will still be included in the over $1000 report